coalib/settings/Section.py

```diff
@@ -65,11 +66,17 @@ def bear_dirs(self):
         bear_dirs = path_list(self.get("bear_dirs", ""))
```

Comment:
No, the `bear_dirs` key is getting escaped here: https://github.com/coala-analyzer/coala/pull/1644/files#diff-41b4f7bdbd660f8c693c7d0304602d79R79

The reason we can't use `glob_list` is because only *certain* bear dirs (the default ones, not the user specified ones) must be escaped. So we can't run `glob_list` directly on the whole `bear_dirs` key.